library/src/amd_detail/rocblaslt/src/tensile_host.cpp

```diff
@@ -526,6 +526,13 @@ namespace
         // set use gradient
         tensileProblem.setUseGradient(is_grad_enabled(prob.epilogue));
 
+        // set AmaxD
+        if(prob.amaxD != nullptr)
+        {
+            tensileProblem.setOutputAmaxD(true);
```

Comment:
better to use tensileProblem.setOutputAmaxD(prob.amaxD != nullptr ? true : false)
It should be fine to set tensileProblem.setAmaxD(compute_type, true) always.

Reply:
Thanks, I see, this makes the style similar to setBias() and setUseBias()
Done